src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -47,10 +50,35 @@
 #include "ui_automatedrssdownloader.h"
 #include "automatedrssdownloader.h"
 
+class AutomatedRssDownloader::DownloadRuleListMatchState
```

Comment:
I can't see the usefulness of this class. Why not just make the QTimer and QSet privatemembers of the `AutomatedRssDownloader` class?
I read the next commit too, but I still don't see the benefit.